app.rb

```diff
@@ -257,6 +279,11 @@ def biometric_comparison_required_value(ial)
       ial == 'biometric-comparison-required'
     end
 
+    def requires_enhanced_ipp?(ial)
+      return unless config.vtr_disabled?
```

Comment:
maybe we add an explicit `false` here to address [Eileen's comment?](https://github.com/18F/identity-oidc-sinatra/pull/161/files#r1612006986)
```suggestion
      return false unless config.vtr_disabled?
```

Reply:
Done. Resolved with commit d9607e4706256f48603e3703c5cf26b5559b0e71